torch/testing/_internal/common_methods_invocations.py

```diff
@@ -419,6 +419,52 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad):
+    test_sizes = [
+        (S,),
+        (S, S),
+        (S, S, S),
+        (S, S, S, S),
+        (0,),
+        (0, 0),
+        (S, 0),
+        (0, S),
+        (0, S, S),
+        (S, 0, S),
+        (0, 0, 0),
+    ]
+    ords = (None, 0, 0.5, 1, 2, 3.5, inf, -0.5, -1, -2, -3.5, -inf)
+    inputs = []
+    is_dtype_half = dtype in [torch.float16, torch.bfloat16]
+
+    def get_all_unique_dim_args(ndim):
+        dim_args = {None}
+        all_dims = list(range(ndim))
+        for num_reduction_dims in range(1, len(test_size) + 1):
+            for dim in combinations(all_dims, num_reduction_dims):
+                dim_args.add(dim)
```

Comment:
I'm getting mypy failures here:
```
Traceback (most recent call last):
  File "test_type_hints.py", line 173, in test_run_mypy
    self.fail(f"mypy failed: {stdout} {stderr}")
AssertionError: mypy failed: torch/testing/_internal/common_methods_invocations.py:445: error: Argument 1 to "add" of "set" has incompatible type "Tuple[int, ...]"; expected "None"  [arg-type]
```

I've tried several different ways of doing this, including:
* Make `dim_args` a list
* Make `dim_args` a set
* Initialize `dim_args` as empty, and append/add `None` later
* Initialize `dim_args` including `None`

None of these avoid the mypy error. I'll keep trying to figure out the right way to do it, but if someone knows what to do, please let me know

Reply:
Looks like I just needed to define `dim_args` as `dim_args: List[Optional[Tuple[int, ...]]] = [None]`